editOrder.php

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="styles/styles.css" rel="stylesheet" type="text/css" />
+    <script type="text/javascript" src="scripts/editorder.js"></script>
+    <link rel="icon" href="assets/icons/SiOverleaf.svg" type="image/icon" />
+    <title>GoToGro | Edit Order</title>
+</head>
+
+<body>
+    <?php
+    include_once("navbar.inc");
+    ?>
+    <div class="editorder" id="editorder">
+        <div>
+            <a class="back-item" href="orderSummery.php">
+                <?php echo file_get_contents("./assets/icons/FiArrowLeft.svg"); ?> Go Back
+            </a>
+        </div>
+        <h1>Edit Order Details</h1>
+        <form method="post" action="updateORDER.php" id="form" id="apply" novalidate="novalidate">
+            <div id="row_1">
+                <div class="inputField">
+                    <label for="product_name">Product Name</label>
+                    <input type="text" name="product_name" id="product_name" disabled>
+                </div>
+                <div class="inputField">
+                    <label for="quantity">Quantity </label>
+                    <input type="number" min="1" max="100" name="quantity" id="quantity" required="required"
```

Comment:
What if someone refunded all of one item?? We don't want to fully delete the item from the Transaction because it is good to have a record that they purchased it in the first place, the minimum value may need to change to accomodate this.

Reply:
Good point, changed it to 0 will be the minimum value, so its recorded in TO but wont affect the Sales Report